### PR TITLE
Omnia/feature/#59 migrate nova resources

### LINKED ISSUE #18: Migrate Nova resources

### PR DESCRIPTION
**Pull request by devjk1**
closes #18

### CODE REVIEW

File: src/Nova/Note.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            MorphTo::make('Noteable')->types([
+                User::class,
+                Contact::class,
+                Customer::class,
+                Order::class,
+                Booking::class,
+                Game::class,
+                Block::class,
+                Slot::class,
+            ]),
```
Comment (drewroberts) [inline src/Nova/Note.php]: @sl0wik What is the best way to handle this?

Comment (joshtorres) [reply src/Nova/Note.php]: @devjk1 added the nova helper in now and is re-running the tests

**Comment by joshtorres** [comment]
@drewroberts should we create the tests or wait until you've merged Patricks changes from these pull requests: https://github.com/tipoff/test-support/pull/9
https://github.com/tipoff/support/pull/46/files

**Comment by drewroberts** [comment]
@joshtorres I just released those packages, you can go ahead and switch to extending the BaseResource in support and setting up the tests.

**Comment by sl0wik** [comment]
Just to confirm - looks good to go.

**Review by drewroberts — APPROVED** [review]
Thank you! 🐽

**Pull request merged**